prow/e2e-suite.sh

```diff
@@ -66,6 +66,7 @@ trap cleanup EXIT
 # use uploaded yaml artifacts rather than the ones generated locally
 DAILY_BUILD=istio-$(echo ${ISTIOCTL_URL} | cut -d '/' -f 6)
 LINUX_DIST_URL=${ISTIOCTL_URL/istioctl/${DAILY_BUILD}-linux.tar.gz}
+DEB_URL=${ISTIOCTL_URL/istioctl/deb}
```

Comment:
please add comment about this weird substitution thing.

Reply:
I rebased from your change so we no longer have to do substitution